chainercv/links/model/ssd/transforms.py

```diff
@@ -0,0 +1,288 @@
+from __future__ import division
+
+import numpy as np
+import random
+import six
+
+from chainercv import utils
+
+
+def random_distort(
+        img,
+        brightness_delta=32,
+        contrast_low=0.5, contrast_high=1.5,
+        saturation_low=0.5, saturation_high=1.5,
+        hue_delta=18):
+    """A data augmentation method for images.
```

Comment:
I think this is too vague.

A color related data augmentation method used in SSD